package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "solid-primitives",
-  "private": true,
+  "private": false,
```

Comment:
Please leave this `true`
It means that the root "package" is not meant to be published to npm if somebody types `npm publish` by mistake 😄